THE TASK 3 PROGRAM "Task3.py" WAS RUN USING PYTHON VERSION 3.7.9
